### PR TITLE
Assign reviewers to PRs

### PR DESCRIPTION
**Pull request by arkodg**
* Uses the CODEOWNERS file to automatically assign reviewers to files to speed up reviews

* As the project progresses, the CODEOWNERS file can be updated to
select a subset of the mantainers as owners for a specific area /
directory.

Signed-off-by: Arko Dasgupta <arko@tetrate.io>

### CODE REVIEW

**Review by danehans — APPROVED** [review]

**Comment by arkodg** [comment]
thanks @danehans , will wait for an mantainer approval from non Tetrands before merging this PR

**Review by youngnick — APPROVED** [review]
Yes, this is a good idea, although I think we should be clear that automatically-assigned reviewers are a *suggestion*, not a requirement. Everyone should feel free to chime in on a review, even if you're not explicitly named.

**Review by lizan — APPROVED** [review]

**Pull request merged**
